docs/content/en/docs-dev/user-guide/managing-application/defining-app-configuration/ecs.md

```diff
@@ -152,7 +152,8 @@ spec:
 ## NOTE
 
 - When you use an ELB for deployments, all listener rules that have the same target groups as configured in app.pipecd.yaml will be controlled.
-- For more information and diagrams, see [Issue#4733 [ECS] Modify ELB listener rules other than defaults without adding config](https://github.com/pipe-cd/pipecd/pull/4733).
+  - For more information and diagrams, see [Issue#4733 [ECS] Modify ELB listener rules other than defaults without adding config](https://github.com/pipe-cd/pipecd/pull/4733).
+- When you use [Service Connect](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/service-connect.html), you cannot use Canary or Blue/Green deployment because Service Connect does not support the external deployment.
```

Comment:
Should be `Not yet supported` instead 😅

Reply:
thanks, I fixed it in [a8e661f](https://github.com/pipe-cd/pipecd/pull/4900/commits/a8e661fff652c900dc66d6ef1553153f656aefce).